flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/aggfunctions/MaxWithRetractAggFunctionTest.java

```diff
@@ -317,6 +318,31 @@ protected List<StringData> getExpectedResults() {
         }
     }
 
+    /** Test for {@link CharType}. */
+    public static final class CharMaxWithRetractAggFunctionTest
+            extends MaxWithRetractAggFunctionTestBase<Character> {
```

Comment:
Why do we need to add this test? CharType and VarCharType both are based on StringData, this test is useless?